gameLogic/GameMap.java

```diff
@@ -9,133 +9,108 @@
 
 public class GameMap implements Drawable {
 	private int currentLevel;
-	private HashMap<Integer, ImageIcon[][]> levelMapping = new HashMap<>();
-	private HashMap<Integer, Rectangle[]> hitRectangleMapping = new HashMap<>();
-
-	public GameMap() {
-		this(1);
-	}
+	private int currentMap;
+	private HashMap<String, ImageIcon[][]> levelMapping = new HashMap<>();
+	private HashMap<String, Rectangle[]> hitRectangleMapping = new HashMap<>();
+
+	public static final int TOTAL_LEVELS = 1;
+	public static final int MAPS_PER_LEVEL = 4;
+	private static final int MAP_WIDTH = 10;
+	private static final int MAP_HEIGHT = 8;
+	private static final int NUM_TILE_TYPES = 10;
+	private static final int TILE_LENGTH = 100;
+
+	private static final String FILEPATH_ROOT = "src/assets/maps/";
+	private static final String FILEPATH_BACKGROUND = "/background";
+	private static final String FILEPATH_PNG = ".png";
 	
-	public GameMap(int level) {
+	public GameMap(int level, int map) {
 		createLevels();
-		currentLevel = level;
+		setStage(level, map);
 	}
-
-
 	
-	public void setMap(int level) {
-		if (level > 0 && level < 4) {
+	public void setStage(int level, int map) {
+		if (level > 0 && level < TOTAL_LEVELS+1 && map > 0 && map < MAPS_PER_LEVEL+1) {
 			currentLevel = level;
+			currentMap = map;
+		} else {
+			currentLevel = 1;
+			currentMap = 1;
 		}
 	}
 
 	public Rectangle[] getCurrentCollisionMap () {
-		return hitRectangleMapping.get(currentLevel);
+		return hitRectangleMapping.get(currentLevel+" "+currentMap);
```

Comment:
Minor comment, but instead of space, can we use - instead? Whitespace can be a potential source of errors